### ISSUE TITLE
Improve the record of the number of downloads 

### ISSUE THREAD

**Issue by zaikunzhang** [pdfo/pdfo.github.io#7]
1. The script that retrieves the number of downloads should be fault tolerant. Use `try ... catch`. If something fails, return 0. 

2. We should not only keep a daily record for the number of downloads for PyPI, but also do the same for GitHub and Anaconda.

**Comment by ragonneau** [comment]
@zaikunzhang For `Conda` and `GitHub`, I recorded the downloads for each version since I do not have access to the downloads per day.

**Comment by zaikunzhang** [comment]
Download on day k = Download up to day k - Download up to day (k - 1). 

This is doable at least for the GitHub downloads. 

Please record either "Download on day k" or "Download up to day k" (both are OK since they are mathematically equivalent) in a list (plain text file) for each k **starting today**. 

Let me rephrase it in case it is still unclear: this list should contain the numbers for each k starting today, not only the number for the current k. We need such data later. This is what I meant a year ago when we began to record the history of the number of downloads, but you did not get it. Hope it is clear this time. 

Thanks.

**Comment by zaikunzhang** [comment]
BTW, check whether the new code works --- the number of downloads today remains the same as yesterday, which is unlikely.

**Comment by ragonneau** [comment]
It is implemented. The fact that the number of downloads did not change is normal since I triggered the counting process manually to be sure that it works on GitHub Actions. My manual trigger and the automated one were finally done the same day.
